### ISSUE TITLE
messes with cls(...) initialization

### ISSUE THREAD

**Issue by samuelcolvin** [koxudaxi/pydantic-pycharm-plugin#65]
The plugin is currently displaying a false warning for the following valid code:

```py
class Foo:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    @classmethod
    def bar(cls):
        return cls(1, 2)
```

**Without the plugin:**
![image](https://user-images.githubusercontent.com/4039449/64610027-35bbdb80-d3c6-11e9-89d7-207f25cef5cb.png)

**With the plugin:**
![image](https://user-images.githubusercontent.com/4039449/64610062-508e5000-d3c6-11e9-9112-83705b7000ef.png)

Running v0.0.18

Here's a more severe example from within pydantic's source code:

![image](https://user-images.githubusercontent.com/4039449/64610240-bb3f8b80-d3c6-11e9-9bd3-3534e5997572.png)

**Comment by koxudaxi** [comment]
Thank you for reporting about the bug.
I have fixed the code.
I will release the version soon.

**Comment by samuelcolvin** [comment]
thank you so much.

**Comment by koxudaxi** [comment]
The fixed version 0.0.19 has been published on JetBrains's market.
Thank you.
